### PR TITLE
Login UI and CORs fixes

### PR DESCRIPTION
**Pull request by Kysumi**

### CODE REVIEW

File: foundationz_backend/src/modules/graphql/auth/auth.ts [hunk 1]
```diff
@@ -3,43 +3,45 @@ import { User } from "@orm/user";
...
+      type: "User",
       args: {
         email: nonNull(stringArg()),
         password: nonNull(stringArg()),
       },
       resolve: async (_, { email, password }, { session }) => {
-        if (session.email) {
-          throw new AuthenticationError(`You are already signed in!`);
+        if (session.userid) {
+          const user = await User.query().findById(session.userid);
```
Comment (Abomy) [inline foundationz_backend/src/modules/graphql/auth/auth.ts:19]: doesn't context already fetch the user? 
Also should probably be using the loader..

File: foundationz_client/src/ui/components/nav/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { Nav } from "grommet";
+import { UserMenu } from "./UserMenu/UserMenu";
+import styled from "styled-components";
+import React from "react";
+import * as Icons from "grommet-icons";
+import { AnchorLink } from "../AnchorLink/AnchorLink";
+
+const Calendar = styled(Icons.Calendar)`
+  margin-left: 10px;
```
Comment (Abomy) [inline foundationz_client/src/ui/components/nav/NavBar.tsx:9]: Shouldn't we be doing separate styling files? 😉

File: foundationz_client/src/ui/components/spinner/FullPageSpinner.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import { Spinner } from "grommet";
+
+const Container = styled.div`
+  position: absolute;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+`;
```
Comment (Abomy) [inline foundationz_client/src/ui/components/spinner/FullPageSpinner.tsx:10]: monster! 😂

File: foundationz_client/tsconfig.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
       "dom.iterable",
       "esnext"
     ],
-    "allowJs": true,
+    "allowJs": false,
```
Comment (Abomy) [inline foundationz_client/tsconfig.json:9]: whys js no invited? D:

File: foundationz_client/src/ui/components/nav/NavBar.tsx (hunk 2 above)
Comment (Kysumi) [reply foundationz_client/src/ui/components/nav/NavBar.tsx:9]: not atm.. keeping it simple till we get settled in trying to generate as few files as possible till I get a good handle on how to work with grommet

Comment (Abomy) [reply foundationz_client/src/ui/components/nav/NavBar.tsx:9]: Sounds good

File: foundationz_backend/src/modules/graphql/auth/auth.ts (hunk 1 above)
Comment (Kysumi) [reply foundationz_backend/src/modules/graphql/auth/auth.ts]: context worked. Take a look

Comment (Abomy) [reply foundationz_backend/src/modules/graphql/auth/auth.ts]: 🙌 solid

**Review by Abomy — APPROVED** [review]

**Pull request merged**